storage/languages/en.json

```diff
@@ -39,11 +39,11 @@
   "seeWhatZCanDo": "See what zephir can do!",
   "support": "Support",
   "team": "Team",
-  "title": "Zephir Programming Language",
+  "title": "Build PHP extensions is easy with Zephir",
```

Comment:
*Building

Feels more appropriate as a subtitle, but ... One nitpick, and I'll merge.